src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -334,18 +357,12 @@ export class WalletOverviewPage {
     this.navCtrl.push(LandingPage);
   }
 
-  /**
-   * Open the settings page
-   */
-  settingsClick(): void {
-    this.navCtrl.push(SettingsGeneralPage);
-  }
-
   /**
    * Whenever the current wallet is changed
    */
   onWalletChanged() {
     this.getWalletBalance(this.currentWallet.publicKey);
+    this.getTransactionHistory(this.currentWallet.publicKey);
```

Comment:
If I'm not mistaken both this call and the call right above it perform a HTTP call. If so a fallback should be programmed in case the HTTP call fails (no internet for example).

Reply:
I've added catches in the promises in the next update so that the promise itself will handle the error. There will be a popup to retry the operation